scripts/prep_changelog.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Script for transforming non-hyperlink changelog entries into ones with
+# hyperlinks using an opionated toolset (https://github.com/chmln/sd and
+# https://github.com/BurntSushi/ripgrep).
```

Comment:
Probably not worth doing now, but is there a way to bundle these in some cargo deps list so users can just `make dependencies` and get these installed for free locally?

Reply:
I should just stop being lazy and make it work with `grep` and `sed`